test/helpers/hasClassBeginningWith.js

```diff
@@ -1,7 +1,7 @@
 import { computed } from '@bigtest/interactor';
 
 export function hasClassBeginningWith(selector, className) {
-  return computed(function () {
+  return computed(function checkHasClassBeginningWith() {
```

Comment:
Is this a common practise to name such functions? IMO it does not bring much value as the function `hasClassBeginningWith` has already described that.

Reply:
@vsoroka07 it is just to satisfy linter.